src/plugin/announcement/Manager/AnnouncementManager.php

```diff
@@ -68,21 +72,13 @@ public function sendMessage(Announcement $announcement, array $users = [])
         $this->om->persist($announcementSend);
         $this->om->flush();
```

Comment:
@Elorfin could we move this logic to the async handler?

Reply:
Yes we can. This is just a log for sent messages. NB. this should be replaced later with generic mail logs.

Reply:
Done, thx :)